.github/workflows/move-work-in-progress-prs.yml

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Move PR To Work In Progress
+on:
+  issue_comment:
+    types: [created]
+    branch:
+      - main
+
+jobs:
+  move-pr-to-wip:
+    if: contains(github.event.comment.body, '/pr mark wip') && github.event.issue.pull_request

```

Comment:
I observe that *anyone* can move a card, not just the PR author. While this is low risk, I think we can easily validate this. After some searching, I think that we want to compare `github.event.comment.user.login` to `github.event.issue.user.login`.